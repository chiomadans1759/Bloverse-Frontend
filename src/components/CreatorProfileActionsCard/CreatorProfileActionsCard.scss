@import 'sass-rem';

#bl-actions{
  .badge {
    background-color: #7b7979;
    width: rem(22px);
    height: rem(16px);
    color: #ffffff;
    border: rem(1px) solid #7b7979;
    border-radius: rem(2px);
  }
}
