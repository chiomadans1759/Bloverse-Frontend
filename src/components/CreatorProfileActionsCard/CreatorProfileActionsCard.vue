<template>
  <main id="bl-actions">
    <div class="bg-white p-4">
      <div class="row mb-3">
        <div class="col-8">
          <h6 class="text-primary"><i class="fal fa-rocket"></i> Posts</h6>
          <p class="text-muted mt-2">Update your information</p>
        </div>
        <div class="col-4 d-flex justify-content-end align-items-center">
          <span class="badge badge-primary">4</span>
        </div>
      </div>
      <div class="row mb-3">
        <div class="col-8">
          <router-link to="/creators/drafts"><h6 class="text-primary"><i class="far fa-edit"></i> Drafts</h6>
          <p class="text-muted mt-2">Update your information</p>
          </router-link>
        </div>
        <div class="col-4 d-flex justify-content-end align-items-center">
          <span class="badge badge-primary">4</span>
        </div>
      </div>
      <div class="row mb-3">
        <div class="col-8">
          <router-link to="/creators/trash">
            <h6 class="text-primary"><i class="fal fa-trash-alt"></i> Trash</h6>
            <p class="text-muted mt-2">Update your information</p>
          </router-link>
        </div>
      </div>
    </div>
  </main>
</template>

<script>
export default {
  
}
</script>

<style lang="scss" scoped>
  @import './CreatorProfileActionsCard';
</style>
