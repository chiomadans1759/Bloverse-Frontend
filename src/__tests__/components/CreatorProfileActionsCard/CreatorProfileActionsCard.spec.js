import { mount } from '@vue/test-utils'
import CreatorProfileActionsCard from '@/components/CreatorProfileActionsCard/CreatorProfileActionsCard.vue';

describe('CreatorActionCard Component', () => {
  it('should render correctly', () => {

    const wrapper = mount(CreatorProfileActionsCard , {});
    expect(wrapper.element).toMatchSnapshot();
  });
});
