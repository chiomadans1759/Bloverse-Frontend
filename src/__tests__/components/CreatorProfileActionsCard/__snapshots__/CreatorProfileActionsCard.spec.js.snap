// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreatorActionCard Component should render correctly 1`] = `
<main
  id="bl-actions"
>
  <div
    class="bg-white p-4"
  >
    <div
      class="row mb-3"
    >
      <div
        class="col-8"
      >
        <h6
          class="text-primary"
        >
          <i
            class="fal fa-rocket"
          />
           Posts
        </h6>
         
        <p
          class="text-muted mt-2"
        >
          Update your information
        </p>
      </div>
       
      <div
        class="col-4 d-flex justify-content-end align-items-center"
      >
        <span
          class="badge badge-primary"
        >
          4
        </span>
      </div>
    </div>
     
    <div
      class="row mb-3"
    >
      <div
        class="col-8"
      >
        <a>
          <h6
            class="text-primary"
          >
            <i
              class="far fa-edit"
            />
             Drafts
          </h6>
           
          <p
            class="text-muted mt-2"
          >
            Update your information
          </p>
        </a>
      </div>
       
      <div
        class="col-4 d-flex justify-content-end align-items-center"
      >
        <span
          class="badge badge-primary"
        >
          4
        </span>
      </div>
    </div>
     
    <div
      class="row mb-3"
    >
      <div
        class="col-8"
      >
        <a>
          <h6
            class="text-primary"
          >
            <i
              class="fal fa-trash-alt"
            />
             Trash
          </h6>
           
          <p
            class="text-muted mt-2"
          >
            Update your information
          </p>
        </a>
      </div>
    </div>
  </div>
</main>
`;
